etc/requirements-elyra.txt

```diff
@@ -0,0 +1,66 @@
+# This is a comprehensive list of python dependencies that Elyra requires to execute Jupyter notebooks.
+# Those that are excluded are transitive dependencies of the included python packages.
+#ansiwrap==0.8.4
+#appdirs==1.4.4
+#appnope==0.1.0
+#async-generator==1.10
+#attrs==19.3.0
+#backcall==0.2.0
+#black==19.10b0
+#bleach==3.1.5
+#certifi==2020.4.5.2
+#chardet==3.0.4
+#click==7.1.2
+#configparser==5.0.0
+#decorator==4.4.2
+#defusedxml==0.6.0
+#entrypoints==0.3
+#idna==2.9
+#importlib-metadata==1.6.1
+ipykernel==5.3.0
+ipython==7.15.0
+ipython-genutils==0.2.0
+#jedi==0.17.0
+#Jinja2==2.11.2
+#jsonschema==3.2.0
+jupyter-client==6.1.3
+jupyter-core==4.6.3
+#MarkupSafe==1.1.1
+minio==5.0.10
+#mistune==0.8.4
+nbclient==0.3.1
+nbconvert==5.6.1
+nbformat==5.0.7
+#nest-asyncio==1.3.3
+#packaging==20.4
+#pandocfilters==1.4.2
+papermill==2.1.1
+#parso==0.7.0
+#pathspec==0.8.0
+#pexpect==4.8.0
+#pickleshare==0.7.5
+#prompt-toolkit==3.0.5
```

Comment:
I wonder if we should include this in our enforcement.  Since I watch the Notebook repo, I got a message regarding issue 4050 (I hesitate to link it since it will add a cross-reference - [here are all notebook issues](https://github.com/jupyter/notebook/issues)).  Then, looking at the IPython repo, I see this [gnarly dependency](https://github.com/ipython/ipython/blob/master/setup.py#L191) which implies IPython is super-sensitive to prompt-toolkit.  As a result, I think we should pin this modules as well.